app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -379,30 +375,37 @@ class Identification extends React.Component {
               <Form.Row className="pt-1">
                 <Form.Group as={Col} md="auto">
                   <Form.Label className="nav-input-label">RACE (SELECT ALL THAT APPLY)</Form.Label>
-                  <Form.Check type="switch" id="white" label="WHITE" checked={this.state.current.patient.white} onChange={this.handleChange} />
+                  <Form.Check type="switch" id="white" label="WHITE" checked={this.state.current.patient.white || false} onChange={this.handleChange} />
```

Comment:
Why the addition of the `|| false`? In order for the `false` to be reached `this.state.current.patient.white` has to be falsey anyway.